tb_plugin/torch_tb_profiler/utils.py

```diff
@@ -24,3 +24,11 @@ def get_logger():
 
 def is_chrome_trace_file(path):
     return consts.WORKER_PATTERN.match(path)
+
+def get_carbon_emit(gpu):
+    watt = consts.WATT_MAP.get(gpu)
```

Comment:
Not sure whether the value of list(profile.gpu_infos.values())[0]['Name'] will exactly match keys in consts.WATT_MAP.
Where do you get the keys of consts.WATT_MAP?